adafruit_displayio_layout/widgets/dial.py

```diff
@@ -0,0 +1,1096 @@
+    :param int minor_tick_length: minor tick length, in pixels
+
+    :param int background_color: background color (RGB tuple
+     or 24-bit hex value), set None for transparent
+
+
+    :param (float,float) anchor_point: (X,Y) values from 0.0 to 1.0 to define the dial's
+     anchor point relative to the dial's bounding box
+    :param (int,int) anchored_position: (x,y) pixel value for the location
+     of the `anchor_point`
+
+
+    **Simple example of dial and moving needle**
+
+    See file: ``examples/displayio_layout_dial_simpletest.py``
+
+
+
+    .. figure:: dial.gif
+       :scale: 100 %
+       :figwidth: 50%
+       :align: center
+       :alt: Diagram of the dial widget with needle in motion.
+
+       This is a diagram of a dial widget with the needle moving from its
+       minimum to maximum positions.
+
+    .. figure:: dial_variables_angles.png
+       :scale: 50 %
+       :figwidth: 70%
+       :align: center
+       :alt: Diagram showing the definition of ``start_angle`` and ``sweep_angle``,
+        both are in units of degrees.
+
+       Diagram showing the definition of ``start_angle`` and ``sweep_angle``,
+       both are in units of degrees.
+
+    .. figure:: dial_variables_min_max_values.png
+       :scale: 50 %
+       :figwidth: 70%
+       :align: center
+       :alt: Diagram showing the defintion of ``min_value`` and ``max_value``.
+
+       Diagram showing the defintion of ``min_value`` and ``max_value``.
+
+    .. figure:: dial_variables_ticks.png
+       :scale: 50 %
+       :figwidth: 70%
+       :align: center
+       :alt: Diagram showing the various parameters for setting the dial labels
+        and major and minor tick marks.
+
+       Diagram showing the various parameters for setting the dial labels
+       and major and minor tick marks.
+    """
+
+    # The dial is a subclass of Group->Widget.
+
+    def __init__(
+        self,
+        width=100,
+        height=100,
+        padding=0,  # keepout amount around border, in pixels
+        sweep_angle=90,  # maximum value is 180 degrees
+        start_angle=None,
+        clip_needle=False,
+        # trims off the needle outside of the dial region, used for sweep_angles < 180
+        needle_width=7,
+        # triangle with this base width, best if this is odd
+        needle_color=0x880000,
+        value=None,
+        value_font=None,
+        display_value=False,
+        value_color=0xFF0000,
+        value_format_string=":0.0f",
+        min_value=0.0,
+        max_value=100.0,
+        anchor_point=None,
+        anchored_position=None,
+        tick_color=0xFFFFFF,
+        major_ticks=5,  # if int, the total number of major ticks
+        major_tick_stroke=3,
+        major_tick_length=10,
+        major_tick_labels=("0", "25", "50", "75", "100"),
+        minor_ticks=5,  # if int, the number of minor ticks per major tick
+        minor_tick_stroke=1,
+        minor_tick_length=5,
+        tick_label_font=None,
```

Comment:
I think this should default to `terminalio.FONT` so if user does not pass anything they will still be able to make a dial.

Reply:
Updated per your suggestions, both for `tick_label_font` and `value_font`.